packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -15,4 +15,22 @@
 	.block-editor-block-inspector__card {
 		margin: 0;
 	}
+
+	// There are to prevent the blocks styles
```

Comment:
Would it make sense to make these styles part of the interface package?

Reply:
Done at https://github.com/WordPress/gutenberg/pull/20530/files?file-filters%5B%5D=.scss#diff-ea6e8e5d17ac512ea81dc2085c71bf43R53 This was... tricky :sweat_smile: 

I tested with the document & block sidebars (edit-post) and global & block sidebars (edit-site). Everything seems to be fine, at least I don't find any gross leak now.